doc/howto/many-executables.rst

```diff
@@ -0,0 +1,150 @@
+How to Organize a Project of Many Executables
+=============================================
+
+Suppose that you're following a tutorial that has these chapters, each with many separate OCaml programs:
+
+- Introducing OCaml
+- Introducing Lwt
+- Introducing Hardcaml
+
+To organize those programs, you can start with a dune project, minimally a
+directory with a ``dune-project`` containing a single line:
+
+.. code:: dune
+
+  (lang dune 3.20)
+
+You can then add executables at will while providing the ``PATH`` argument, the
+argument after the executable name:
```

Comment:
I find this hard to read. Maybe because `PATH` makes my brain think of the `$PATH` shell variable.

Maybe something like

```suggestion
You can add executables by using `dune init exe` and providing the name of the executable.
```